frontend/chat-plugin/BUILD

```diff
@@ -8,6 +8,9 @@ load("@io_bazel_rules_docker//container:container.bzl", "container_image")
 
 module_deps = [
     "//lib/typescript/types",
+    "//lib/typescript/httpclient",
+    "//lib/typescript/render",
+    "//lib/typescript/services",
```

Comment:
the name is too generic here and right now it contains only dates. In general, I'd rather a few small specialised libraries than a giant services/util library. So I suggest we call this `lib/typescript/dates`

Reply:
that way you can also have `lib/typescript/dates/format.ts` which reads kind of nice